ThreadGroup/Views/TGSelectDeviceStepView.m

```diff
@@ -65,6 +65,10 @@ - (void)commonInit {
 
 #pragma mark - Public
 
+- (void)setTopBarHidden:(BOOL)hidden {
+    self.topBar.hidden = hidden;
+}
```

Comment:
I think you could just call `self.topBar.hidden = YES` or `self.topBar.hidden = NO`